src/support/thread_group.c

```diff
@@ -175,14 +175,10 @@ __thread_group_resize(WT_SESSION_IMPL *session, WT_THREAD_GROUP *group, uint32_t
      */
     for (i = group->max; i < new_max; i++) {
         WT_ERR(__wt_calloc_one(session, &thread));
-        /*
-         * Threads get their own session and hs table cursor (if the hs table is open).
-         */
+        /* Threads get their own session */
```

Comment:
Should have a full stop at end of comment.

Reply:
Fixed.